.github/workflows/check-cargo-check-runtimes.yml

```
@@ -120,6 +124,26 @@ jobs:
         with:
           root: cumulus/parachains/runtimes/testing
 
+  cargo-check-each-crate-macos:
```

```suggestion
  cargo-check-all-crate-macos:
```

This is not using the script. Generally we only want to check that everything would compile on macos and using `cargo check --workspace` for that is fine.